cypress/support/fragments/service_point/switchServicePoint.js

```diff
@@ -26,7 +41,7 @@ export default {
       Button({ id: 'accordion-toggle-button-permissions' }).click(),
       Button({ id: 'clickable-add-permission' }).click(),
       Modal({ id: 'permissions-modal' }).find(TextField({ type: 'search' })).fillIn(permissions.uiCheckinAll.gui),
-      Modal({ id: 'permissions-modal' }).find(Button('Search')).click(),
+      Modal({ id: 'permissions-modal' }).find(searchButton).click(),
```

Comment:
I think also need remove MOdal to the const

Reply:
Done